visualization/app/codeCharta/ui/customConfigs/customConfigList/customConfigList.component.ts

```diff
@@ -0,0 +1,32 @@
+import "./customConfigList.component.scss"
+import { Component, Inject, OnDestroy, OnInit } from "@angular/core"
+import { CustomConfigGroups } from "../../../util/customConfigHelper"
+import { CustomConfigFileStateConnector } from "../customConfigFileStateConnector"
+import { Subscription } from "rxjs"
+import { CustomConfigHelperService } from "../customConfigHelper.service"
+
+@Component({
+	template: require("./customConfigList.component.html")
+})
+export class CustomConfigListComponent implements OnInit, OnDestroy {
+	customConfigFileStateConnector: CustomConfigFileStateConnector
+	dropDownCustomConfigItemGroups: CustomConfigGroups
+	isCollapsed = true
+	subscription: Subscription
+
+	constructor(@Inject(CustomConfigHelperService) private customConfigService: CustomConfigHelperService) {}
+
+	ngOnInit() {
```

Comment:
We can subscribe in the template / use the async pipe and don't have to do it manually like

`<mat-dialog-content class="content" *ngIf="customConfigService.customConfigItemGroups$ | async; let dropDownCustomConfigItemGroups">
`